packages/tx/docs/classes/BlobEIP4844Transaction.md

```diff
@@ -0,0 +1,997 @@
+[@ethereumjs/tx](../README.md) / BlobEIP4844Transaction
+
+# Class: BlobEIP4844Transaction
+
+Typed transaction with a new gas fee market mechanism for transactions that include "blobs" of data
+
+- TransactionType: 5
```

Comment:
hasn't this been changed to 3?

Reply:
yep: https://eips.ethereum.org/EIPS/eip-4844

Reply:
yes, its basically in our `develop-7` branch, `master` currently is just maintenance and we will hopefully shift to `develop-v7` soon